lib/proxy/request.rb

```diff
@@ -26,7 +26,7 @@ def uri(path)
 
     def add_headers(req, headers = {})
       req.add_field('Accept', 'application/json,version=2')
-      req.content_type = headers["Content-Type"] || 'application/json'
+      req.content_type = headers.delete("Content-Type") || 'application/json'
```

Comment:
without this, `req.add_field` on line 31 would add it again because we iterate over all headers, that caused the test problems